src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -461,6 +461,8 @@ public static void run(Extension extension) throws IOException {
         logger.info("Starting extension " + extension.getExtensionSettings().getExtensionName());
         @SuppressWarnings("unused")
         ExtensionsRunner runner = new ExtensionsRunner(extension);
+        // initialize the transport service
+        new NettyTransport(runner).initializeExtensionTransportService(runner.getSettings(), new ThreadPool(runner.getSettings()), runner);
```

Comment:
Let's create an object of `NettyTransport` and then invoke `initializeExtensionTransportService`. Looks clean :)

Reply:
Blame me for that combo. :)